project/webapp/templates/thread.html

```diff
@@ -53,6 +53,16 @@
 <body>
     {% include "partials/utils/global_nav.html" %}
     <div id="thread"></div>
+
+    <div id="fb-root"></div>
+    <script>(function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = 'https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v3.0';
```

Comment:
As requested during our weekly meeting, please do not embed third party code in our templates, particularly from Facebook. Facebook uses these beacons to track and profile people. Using these beacons in our website makes us complicit with the violation of people's privacy.